src/better-shifting.cow

```diff
@@ -422,10 +422,191 @@ or by a scalar shift amount which applies to all elements in the \tcode{simd::ve
 
 The following changes are relative to \ref[N5008].
 
-\Btodo{
-This.
+\Bnote{
+The \tcode{shl} and \tcode{shr} functions should immediately precede the
+\tcode{rotl} and \tcode{rotr} functions in the wording. Therefore, if this paper
+and \ref[P3764] are adopted in the same meeting, the \tcode{shl} and \tcode{shr}
+functions should be inserted between the \tcode{msb_to_mask} and \tcode{rotl}
+functions.
 }
 
+\h3{[bit]}
+
+In \eelis{bit.syn}, change the synopsis as follows:
+
+\Bdiff{
+\itemdecl{
+namespace std {
+  \html{[\N{HORIZONTAL ELLIPSIS}]}
+
+\ins{  // \serif{[bit.shift], shifting}
+  template<class T>
+    constexpr T shl(T x, int s) noexcept;
+  template<class T>
+    constexpr T shr(T x, int s) noexcept;
+}
```

Comment:
```suggestion
    constexpr T shr(T x, int s) noexcept;}
```
whitespace is preserved as is in this context, so putting the closing brace on another line would introduce two extra lines.